src/main/java/org/skypro/exam/controller/ExamController.java

```diff
@@ -0,0 +1,24 @@
+package org.skypro.exam.controller;
+
+import org.skypro.exam.service.ExaminerServiceImpl;
+import org.skypro.exam.service.QuestionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping
+public class ExamController {
+    private final ExaminerServiceImpl examinerService;
+
+    @Autowired
+    public ExamController(ExaminerServiceImpl examinerService) {
```

Comment:
используем плюсы полиморфизма
```suggestion
    public ExamController(ExaminerService examinerService) {
```